config/routes.rb

```diff
@@ -89,8 +89,11 @@
   get '/patients/:patient_submission_token/:lang/:unique_identifier', to: 'assessments#new', as: :new_patient_assessment_jurisdiction_lang
   get '/report/patients/:patient_submission_token/:unique_identifier', to: 'assessments#new', as: :new_patient_assessment_jurisdiction_report
   get '/report/patients/:patient_submission_token/:lang/:unique_identifier', to: 'assessments#new', as: :new_patient_assessment_jurisdiction_report_lang
+  get '/r/:patient_submission_token/:unique_identifier/:lang', to: 'assessments#new', as: :new_patient_assessment_jurisdiction_lang_no_initials
```

Comment:
So is the idea of supporting this route for the case where there are no initials for a given patient?

Reply:
yeah, or if the initials or age is blank, the constructed route will end up routing to this

Reply:
Gotcha, might be good to add a test for this (unless there is one and I'm just missing it).

Reply:
good point